flutter/news/lib/features/frontpage/domain/entities/base_news_response.dart

```diff
@@ -1,23 +1,15 @@
-import 'package:equatable/equatable.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
 import 'package:news/features/frontpage/domain/entities/article.dart';
-import 'package:json_annotation/json_annotation.dart';
+part 'base_news_response.freezed.dart';
 part 'base_news_response.g.dart';
 
-@JsonSerializable()
-class BaseNewsResponse extends Equatable {
-  final String status;
-  final int totalResults;
-  final List<Article> articles;
-
-  const BaseNewsResponse(
-      {required this.status,
-      required this.totalResults,
-      required this.articles});
+@freezed
+class BaseNewsResponse with _$BaseNewsResponse {
+  const factory BaseNewsResponse(
+      {required String status,
+      required int totalResults,
+      required List<Article> articles}) = _BaseNewsResponse;
```

Comment:
```suggestion
      required List<Article> articles},
  ) = _BaseNewsResponse;
```

Reply:
I will try enforcing the usage of commas later on a different PR